docs/api/cookies.md

```diff
@@ -120,3 +120,7 @@ Removes the cookies matching `url` and `name`
 Returns `Promise<void>` - A promise which resolves when the cookie store has been flushed
 
 Writes any unwritten cookies data to disk.
+
+Cookies written by any method will not be written to disk immediately, but will be written every 30 seconds or when the number of cookies exceeds 512.
```

Comment:
> when the number of cookies exceeds 512.

From the link in [my other comment](https://github.com/electron/electron/pull/37572/files#r1146879718), I'm not sure if "512 operations" equals 512 cookies. Do you have an example or some docs that shows this?

Reply:
Ok, I'll try to write some examples later on

Reply:
Let me revise the instructions

Reply:
Do you think it is possible now

Reply:
@erickzhao